ddtrace/debugging/_exception/replay.py

```diff
@@ -188,7 +190,9 @@ def on_span_exception(
                         )
 
                         # Capture
-                        snapshot.line()
+                        snapshot.line(ChainMap(frame.f_locals, frame.f_globals))
```

Comment:
we call this ChainMap multiple locations - I think it better to have an util method e.g; createScopeFromFrame(frame) -> to avoid problems down the line if we decide to limit or remove globals...

or maybe line() should just get the frame object instead.

Reply:
The arguments are all different so I don't think we can make a single helper in this case 🙁